ariadne/cardano/src/Ariadne/Wallet/Cardano/Kernel/DB/Util/IxSet.hs

```diff
@@ -105,6 +105,9 @@ newtype IxSet a = WrapIxSet {
 instance Show a => Show (IxSet a) where
     show = show . map unwrapOrdByPrimKey . IxSet.toList . unwrapIxSet
 
+instance (Eq a, Indexable a) => Eq (IxSet a) where
```

Comment:
Oh, It can be derived. I did not know about the standalone deriving instance, that is why I did it by hands. I 've just read [docs about it](https://downloads.haskell.org/~ghc/latest/docs/html/users_guide/glasgow_exts.html#stand-alone-deriving-declarations) and derived the instance instead and removed written by hands.